### ISSUE TITLE
member-ordering rule doesn't allow to manage order of abstract members

### ISSUE THREAD

**Issue by kapke** [palantir/tslint#3237]
### Bug Report

- __TSLint version__: 5.7.0
- __TypeScript version__: 2.5.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
abstract class Foo {
    protected abstract hello();

    public getFoo () {
        return 'foo';
    }
}
```

with `tslint.json` configuration:

Any with enabled member ordering

#### Actual behavior
There is no way to treat abstract members differently so enabling `member-ordering` rule forces to move all abstract members to a proper group.

#### Expected behavior
I believe that most people put abstract members somewhere in the top of the class to know what has to be implemented in derived class. There should be a way to declare where should abstract members be. 
1. By declaring other types of members: `private-abstract-method`, `protected-abstract-method`, `public-abstract-method`.
2. By adding support for a flag like `alphabetize` (BTW the documentation doesn't say it clear how to use it and schema doesn't define it too). It could be named like `abstract-first`

**Comment by JoshuaKGoldberg** [comment]
☠️ *TSLint's time has come!* ☠️ 

TSLint is no longer accepting most feature requests per #4534. See **[typescript-eslint.io](https://typescript-eslint.io)** for the new, shiny way to lint your TypeScript code with **ESLint**. ✨ 

It was a pleasure open sourcing with you all!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
